src/language-css/print/parenthesized-value-group.js

```diff
@@ -78,7 +81,13 @@ function printParenthesizedValueGroup(path, options, print) {
 
   if (!node.open) {
     const forceHardLine = shouldBreakList(path);
-    const parts = join([",", forceHardLine ? hardline : line], groupDocs);
+    assertDocArray(groupDocs);
+    const parts = join(
+      forceHardLine ? hardline : line,
+      groupDocs.map((doc, i) =>
+        i === groupDocs.length - 1 ? doc : [doc, ","],
+      ),
+    );
```

Comment:
Maybe we can use a `chunk()` unitity

```js
join(forceHardLine ? hardline : line, chunk(join(',', groupDocs), 2))
```

But still not look good..

Reply:
I can't find `chunk()` utility...
I searched `function chunk` over the repository and read `builders.js` manually.

Reply:
We don't have it.

Reply:
So you mean we can write a new utility, right?
I'll write it in `parenthesized-value-group.js`. Ask me to move it if you supposed another file.